test/functional/api/buffer_spec.lua

```diff
@@ -1,5 +1,5 @@
 -- Sanity checks for buffer_* API calls via msgpack-rpc
-local helpers = require('test.functional.helpers')
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
Though I did not try to mess with busted internals, only checked where `after_each` is available (it is not when importing module and it is not in a helper script).